app/bundles/ReportBundle/Tests/Controller/ReportControllerFunctionalTest.php

```diff
@@ -53,6 +54,44 @@ public function testMostVisitedPagesReport(): void
         Assert::assertTrue($this->client->getResponse()->isOk());
     }
 
+    public function testReportTableOrderColumn(): void
+    {
+        $page  = $this->createPage('test page 1', 15);
+        $page2 = $this->createPage('test page 2', 9);
+        $page3 = $this->createPage('test page 3', 30);
+
+        $this->createHit($page);
+        $this->createHit($page2);
+        $this->createHit($page3);
```

Comment:
```suggestion
```
This is probably not needed, it creates only one hit per page

Reply:
Without this there are no report results.